src/Microsoft.Health.Dicom.Core/Features/Validation/ElementMinimumValidator.cs

```diff
@@ -12,24 +12,23 @@ namespace Microsoft.Health.Dicom.Core.Features.Validation
 {
     public class ElementMinimumValidator : IElementMinimumValidator
     {
-
         private static readonly IReadOnlyDictionary<DicomVR, IElementValidation> Validations = new Dictionary<DicomVR, IElementValidation>
         {
             { DicomVR.AE, new ElementMaxLengthValidation(16) },
             { DicomVR.AS, new ElementRequiredLengthValidation(4) },
             { DicomVR.CS, new ElementMaxLengthValidation(16) },
             { DicomVR.DA, new DateValidation() },
-            { DicomVR.DS, new ElementMaxLengthValidation(16) },
-            { DicomVR.DT, new DateTimeValidation() },
-            { DicomVR.FL, new ElementRequiredLengthValidation(4) },
+            { DicomVR.DS, new EncodedStringElementValidation() },
```

Comment:
Should we remove `DS` as we don't support it now?

Reply:
We totally can, as it may be confusing to leave it around

Reply:
This is pending, right?